mii/config.py

```diff
@@ -123,4 +126,25 @@ class LoadBalancerConfig(BaseModel):
 
     class Config:
         validate_all = True
-        validate_assignment = True
+
+    validate_assignment = True
+
+
+class DeploymentConfig(BaseModel):
+    deployment_name: str = Field(alias="DEPLOYMENT_NAME_KEY")
```

Comment:
I'm a little bit confused about these aliases. I know we discussed a change here last week, but I think there may have been some miscommunication.